easybuild/tools/systemtools.py

```diff
@@ -36,11 +36,7 @@
 import sys
 from socket import gethostname
 from vsc.utils import fancylogger
-try:
-    # this import fails with Python 2.4 because it requires the ctypes module (only in Python 2.5+)
-    from vsc.utils.affinity import sched_getaffinity
-except ImportError:
-    pass
+from vsc.utils.affinity import sched_getaffinity
```

Comment:
oooh, this uses ctypes and assumes stuff about glibc. careful with this one on non-linux.